frontend/src/components/eMIB/EditEmail.jsx

```diff
@@ -350,20 +360,23 @@ class EditEmail extends Component {
                 )}
               </label>
               <OverlayTrigger
-                trigger="focus"
+                trigger="click"
                 placement="right"
                 overlay={
-                  <Popover id="reasons-for-action-tooltip">
+                  <Popover>
                     <div>
                       <p>{LOCALIZE.emibTest.inboxPage.addEmailResponse.emailResponseTooltip}</p>
                     </div>
                   </Popover>
                 }
               >
                 <Button
+                  id="your-response-tooltip-button"
+                  tabIndex="-1"
                   aria-label={LOCALIZE.ariaLabel.emailResponseTooltip}
                   style={styles.tooltipButton}
                   variant="link"
+                  onBlur={this.triggerResponseTooltipClick}
```

Comment:
...remove the `onBlur` action and it'll do the same behavior.

Reply:
removed in ea161a4